tests/adapters/test_parameter_tree.py

```diff
@@ -6,6 +6,7 @@
 from copy import deepcopy
 
 import pytest
+import logging  # dont leave this here
```

Comment:
See trailing comment - there are also `logging.debug` calls still in tests - most commented out but one still active, which should be removed